hardware.inc

```diff
@@ -457,226 +405,230 @@ AUDENA_ON    EQU %10000000
 AUDENA_OFF   EQU %00000000  ; sets all audio regs to 0!
 
 
-;***************************************************************************
-;*
-;* SoundChannel #1 registers
-;*
-;***************************************************************************
+; --
+; -- LCDC ($FF40)
+; -- LCD Control (R/W)
+; --
+rLCDC EQU $FF40
+
+LCDCF_OFF     EQU %00000000 ; LCD Control Operation
+LCDCF_ON      EQU %10000000 ; LCD Control Operation
+LCDCF_WIN9800 EQU %00000000 ; Window Tile Map Display Select
+LCDCF_WIN9C00 EQU %01000000 ; Window Tile Map Display Select
+LCDCF_WINOFF  EQU %00000000 ; Window Display
+LCDCF_WINON   EQU %00100000 ; Window Display
+LCDCF_BG8800  EQU %00000000 ; BG & Window Tile Data Select
+LCDCF_BG8000  EQU %00010000 ; BG & Window Tile Data Select
+LCDCF_BG9800  EQU %00000000 ; BG Tile Map Display Select
+LCDCF_BG9C00  EQU %00001000 ; BG Tile Map Display Select
+LCDCF_OBJ8    EQU %00000000 ; OBJ Construction
+LCDCF_OBJ16   EQU %00000100 ; OBJ Construction
+LCDCF_OBJOFF  EQU %00000000 ; OBJ Display
+LCDCF_OBJON   EQU %00000010 ; OBJ Display
+LCDCF_BGOFF   EQU %00000000 ; BG Display
+LCDCF_BGON    EQU %00000001 ; BG Display
+; "Window Character Data Select" follows BG
+
+
+; --
+; -- STAT ($FF41)
+; -- LCDC Status   (R/W)
+; --
+rSTAT EQU $FF41
+
+STATF_LYC     EQU  %01000000 ; LYC=LY Coincidence (Selectable)
+STATF_MODE10  EQU  %00100000 ; Mode 10
+STATF_MODE01  EQU  %00010000 ; Mode 01 (V-Blank)
+STATF_MODE00  EQU  %00001000 ; Mode 00 (H-Blank)
+STATF_LYCF    EQU  %00000100 ; Coincidence Flag
+STATF_HB      EQU  %00000000 ; H-Blank

```

Comment:
I have a personal preference for "VBL" and "HBL", but that's probably to taste.

Reply:
I don't mind either way; changing to H/VBL.